tests/c/varargs_inlined.c

```diff
@@ -0,0 +1,63 @@
+// Run-time:
+//   env-var: YKD_PRINT_IR=aot
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     jit-state: start-tracing
+//     i=1
+//     jit-state: stop-tracing
+//     --- Begin aot ---
+//     ...
+//     call void @llvm.va_start...
+//     ...
+//     call void @llvm.va_end...
+//     ...
+//     --- End aot ---
+//     i=1
+//     jit-state: enter-jit-code
+//     i=1
+//     jit-state: exit-jit-code
+//     jit-state: enter-jit-code
+//     i=1
+//     jit-state: enter-stopgap
+//     jit-state: exit-stopgap
+
+// Check that basic trace compilation works.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <yk.h>
+#include <yk_testing.h>
+
+int varargfunc(int len, ...) {
+  int acc = 0;
+  va_list argp;
+  va_start(argp, len);
+  int arg = va_arg(argp, int);
+  acc += arg;
+  va_end(argp);
+  return acc;
+}
+
+int foo(int argc) { return varargfunc(3, argc * 1, 2, 3); }
```

Comment:
argc * 1

Reply:
Fixed in [015fd3b](https://github.com/ykjit/yk/pull/568/commits/015fd3be20fce54242c54cc395f56466273fc832)